test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -661,7 +663,7 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 					"kernel.sched_rt_runtime_us":    "950000",
 					"vm.stat_interval":              "10",
 				}
-				kernelParameters := []string{"processor.max_cstate=1", "intel_idle.max_cstate=0"}
+				kernelParameters := []string{"processor.max_cstate=1", "intel_idle.max_cstate=0", "intel_pstate=disable"}
```

Comment:
is failing due to rebase with this fix https://github.com/openshift/cluster-node-tuning-operator/pull/480, right?

 ```
      [test_id:50992][crit:high][vendor:cnf-qe@redhat.com][level:acceptance]should update kernel arguments and tuned accordingly [It]
      /go/src/github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go:639
      Expected
              
          <string>: BOOT_IMAGE=(hd0,gpt3)/ostree/rhcos-722b79b30961f9836d5f14e4a3b509b8404f8bfeddb1837db2100780f6d80f82/vmlinuz-4.18.0-372.26.1.el8_6.x86_64 ostree=/ostree/boot.0/rhcos/722b79b30961f9836d5f14e4a3b509b8404f8bfeddb1837db2100780f6d80f82/0 ignition.platform.id=gcp console=tty0 console=ttyS0,115200n8 root=UUID=d2dc0c64-1fb9-4f74-9121-2a015d3d9b92 rw rootflags=prjquota boot=UUID=904ab62b-2b2c-408f-9439-40b0e76eef31 skew_tick=1 nohz=on rcu_nocbs=1-3 tuned.non_isolcpus=00000001 systemd.cpu_affinity=0 intel_iommu=on iommu=pt isolcpus=managed_irq,1-3 processor.max_cstate=1 intel_idle.max_cstate=0 default_hugepagesz=1G hugepagesz=2M hugepages=128
          
      to contain substring
          <string>: intel_pstate=disable 
```

Reply:
@marioferh i don't know why this test case is failing in this PR and not on any other PR.  I am not touching workload hints in this PR.

Reply:
@marioferh i apologize, i didn't notice i inadvertently added this while testing something related to workload hints manually.